osx/qmk_toolbox/Base.lproj/MainMenu.xib

```diff
@@ -301,7 +301,7 @@
                             <subviews>
                                 <comboBox verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="j37-hH-gaF">
                                     <rect key="frame" x="7" y="5" width="149" height="25"/>
-                                    <comboBoxCell key="cell" scrollable="YES" lineBreakMode="clipping" selectable="YES" editable="YES" enabled="NO" sendsActionOnEndEditing="YES" borderStyle="bezel" placeholderString="" drawsBackground="YES" completes="NO" numberOfVisibleItems="20" id="5mG-K9-oUt">
+                                    <comboBoxCell key="cell" scrollable="YES" lineBreakMode="clipping" selectable="YES" enabled="NO" sendsActionOnEndEditing="YES" borderStyle="bezel" placeholderString="--- Select a keyboard to download ---" drawsBackground="YES" completes="NO" numberOfVisibleItems="20" id="5mG-K9-oUt">
```

Comment:
```suggestion
                                    <comboBoxCell key="cell" scrollable="YES" lineBreakMode="clipping" selectable="YES" editable="YES" enabled="NO" sendsActionOnEndEditing="YES" borderStyle="bezel" placeholderString="Select a keyboard to download" drawsBackground="YES" completes="NO" numberOfVisibleItems="20" id="5mG-K9-oUt">
```

I actually think we *should* keep it editable. The list is so long that autocomplete would probably be a good idea. We just need to tick the Autocompletes checkbox for the NSComboBox:
<img width="358" alt="image" src="https://user-images.githubusercontent.com/4781841/77485707-edf03380-6e81-11ea-8102-22e886cf0ea3.png">

Reply:
Cool, didn't realize autocomplete for both Windows and Mac were built-in to the widgets.